jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -266,6 +266,9 @@ public class CategoryAttributesDetailView extends StandardDetailView<CategoryAtt
     private JmixButton editEnumerationBtn;
     private boolean isRefreshing = false;
 
+    @ViewComponent
```

Comment:
Please sort fields - this field should be with other ViewComponents fields - TypedTextField nearby other text fields

Reply:
![Uploading image.png…]()